openpype/modules/slack/plugins/publish/integrate_slack_api.py

```diff
@@ -62,8 +62,16 @@ def process(self, instance):
                     client = SlackPython3Operations(token, self.log)
 
                 if "@" in message:
-                    if users is None:
+                    cache_key = "__cache_ids"
```

Comment:
```suggestion
                    cache_key = "__cache_ids"
```

Note that the context data is accessible by all plug-ins. It might be useful to avoid clashing with anything else that'll try to cache ids or alike like this.

Would: `self.__class__.__name__ + "__cache_ids` be a good candidate? or at least `__slack_user_cache`?